low-level/files/opt/seagate/sspl/setup/sspl_test.py

```diff
@@ -196,21 +218,27 @@ def process(self):
             Conf.save(SSPL_CONFIG_INDEX)
             shutil.copyfile(sspl_test_backup, sspl_test_file_path)
             if rc != 0:
-                raise TestException("%s - ERROR: %s - CMD %s" % (self.name, error, CMD))
+                raise TestException(
+                    "%s - ERROR: %s - CMD %s" % (self.name, error, CMD))
 
             print('Restarting the SSPL service..')
             CMD = "systemctl restart sspl-ll"
             try:
                 SimpleProcess(CMD).run(realtime_output=True)
             except Exception as error:
-                raise TestException("Error occurred while executing sspl tests: %s" %error)
+                raise TestException(
+                    "Error occurred while executing sspl tests: %s" % error)
+        elif self.plan in NOT_IMPLEMENTED_TEST_PLANS:
+            print("TEST SKIPPED: No tests included for %s test plan." % (
```

Comment:
"Tests skipped, test plan %s not applicable for SSPL .."

And make we return success for these skipped tests

Reply:
Done.